flake.nix

```diff
@@ -1,15 +1,37 @@
 {
   description = "ghc-dev";
+
+  inputs = {
+    terranix.url = "github:terranix/terranix";
+    terranix.inputs.nixpkgs.follows = "nixpkgs";
+  };
+
   outputs =
-    { self, nixpkgs }:
+    { self, nixpkgs, terranix }:
     let
       system = "x86_64-linux";
       ghc = "ghc92";
       pkgs = nixpkgs.legacyPackages.${system};
+      lib = nixpkgs.lib;
       haskellPackages =
         pkgs.haskell.packages.${ghc}.extend(hself: hsuper: {
           ghc-dev-webgen = haskellPackages.callCabal2nix "ghc-dev-webgen" "${self}/src/" {};
         });
+      terraform = pkgs.terraform;
+      terraformConfiguration = terranix.lib.terranixConfiguration {
+        inherit system;
+        modules = [ ./deployment/main.nix ];
+      };
+      mkTfApp = command: {
```

Comment:
These helpers are available in `serokell.nix`, let's reuse them instead of copy-pasting

Reply:
adjusted